assets/webhook_deployment.yaml

```diff
@@ -0,0 +1,46 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: csi-snapshot-webhook
+  namespace: openshift-cluster-storage-operator
+spec:
+  serviceName: "csi-snapshot-webhook"
+  replicas: 1
+  selector:
+    matchLabels:
+      app: csi-snapshot-webhook
+  template:
+    metadata:
+      labels:
+        app: csi-snapshot-webhook
+    spec:
+      containers:
+      - name: webhook
+        image: ${WEBHOOK_IMAGE}
+        args:
+          - --tls-cert-file=/etc/snapshot-validation-webhook/certs/tls.crt
+          - --tls-private-key-file=/etc/snapshot-validation-webhook/certs/tls.key
+          - "--v=${LOG_LEVEL}"
+        ports:
+        - containerPort: 443
+        volumeMounts:
+          - name: certs
+            mountPath: /etc/snapshot-validation-webhook/certs
+            readOnly: true
+        imagePullPolicy: IfNotPresent
+        resources:
+          requests:
+            cpu: 10m
+      serviceAccountName: csi-snapshot-controller-operator
```

Comment:
Is this what we want?

Reply:
The webhook does not talk to Kubernetes API at all. Should it have a separate service account?

Reply:
Should this webhook have a SA at all?

The multus webhook doesn't need one: https://github.com/openshift/multus-cni/blob/master/deployment/webhook/deployment.yaml

Reply:
Removed the service account, it will get the default one from the namespace.